### ISSUE TITLE
User can't start Teeworlds, gets "VCRUNTIME140.dll not found" error

### ISSUE THREAD

**Issue by Dune-jr** [teeworlds/teeworlds#2082]
Got a support request for a user that gets "System Error-----...VCRUNTIME140.dll was not found" when running Teeworlds. 

- Windows 10
- first-time install, so I assume no settings.cfg. 
- tried with Steam and no Steam
- tried with win64 and win32

Anything else one can do to troubleshoot?

**Comment by heinrich5991** [comment]
Sounds like the MSVS redistributable needs to be installed. We should probably ship them with the Teeworlds release if we're allowed to, or make a note that they should be installed (not really userfriendly, skip that), or we could statically link to the mingw libraries.

**Comment by LordSk** [comment]
We usually don't have that issue I think. I built the windows x64 0.7.2 binary if you remember, using msvc 2017. Oy builds it differently.

**Comment by axblk** [comment]
The freetype dll requires the MSVC redistributable. It is statically linked into the teeworlds binary I think (at least for bam).

**Comment by Dune-jr** [comment]
The user claims to have tried 64-bit Windows, @LordSk. Are you sure you did the 0.7.2 win64 release though? I think @oy did it

**Comment by Dune-jr** [comment]
Installing the MSVC redistributable solved that:

> the bug i had "[teeworlds/teeworlds] Issue opened: #2082 User can't start Teeworlds, gets "VCRUNTIME140.dll not found" error" was solved by downloading MSVC redistributable. Thank you very much for the support :^)

It's not a good solution, though

**Comment by oy** [comment]
If that problem is linked to the freetype dll like @axblk mentioned someone having the issue might try the old freetype dll from the 0.6 branch https://github.com/teeworlds/teeworlds/blob/0.6/other/freetype/windows/lib32/freetype.dll. Never saw people reporting that issue for 0.6 versions?

**Comment by Dune-jr** [comment]
It's right I don't think anyone had that issue in 0.6.
This user claims he already has the MSVC redistributable: 

> I already installed the Microsoft Visual C++ Redistributable Packages for Visual Studio;
My OS is windows 10 Home and i tried to install teewords from steam before and from teeworlds.com after.

https://steamcommunity.com/app/380840/discussions/0/1680315447987473432/

It's a bit difficult to try that fix because all the people that are hitting on this bug seem to be casual players

**Comment by axblk** [comment]
According to dumpbin the 0.6 freetype dll depends on the VS 2010 redistributable.

```
  Image has the following dependencies:

    MSVCR100.dll
    KERNEL32.dll
```

Output for the 0.7 dll:
```
  Image has the following dependencies:

    VCRUNTIME140.dll
    api-ms-win-crt-convert-l1-1-0.dll
    api-ms-win-crt-string-l1-1-0.dll
    api-ms-win-crt-utility-l1-1-0.dll
    api-ms-win-crt-environment-l1-1-0.dll
    api-ms-win-crt-stdio-l1-1-0.dll
    api-ms-win-crt-heap-l1-1-0.dll
    api-ms-win-crt-runtime-l1-1-0.dll
    KERNEL32.dll
```

The one ddnet is using does not depend on any VS redistributable. They compiled it themselves with mingw I think.

**Comment by oy** [comment]
Should be fixed by aa0bf33ad9149b7d86f9aa634c592c1a5712f3dc

**Comment by Dune-jr** [comment]
Neat!
